docs/windows/latest-supported-vc-redist.md

```diff
@@ -0,0 +1,94 @@
+Downloaded additional versions and languages form [Update for Visual C++ 2013 Redistributable Package](https://support.microsoft.com/en-us/topic/update-for-visual-c-2013-redistributable-package-d8ccd6a5-4e26-c290-517b-8da6cfdf4f10) or from [My.VisualStudio.com](https://my.visualstudio.com/).
+
+| Architecture |   Version    |                                                    Link |
+| ------------ | :----------: | ------------------------------------------------------: |
+| X86          | 12.0.40664.0 | [vcredist_x86.exe](https://aka.ms/highdpimfc2013x86enu) |
+| X64          | 12.0.40664.0 | [vcredist_x64.exe](https://aka.ms/highdpimfc2013x64enu) |
+
+### Other versions
+
+- [Multibyte MFC Library for Visual Studio 2013](https://my.visualstudio.com/Downloads?pid=1430). This add-on for Visual Studio 2013 contains the multibyte character set (MBCS) version of the Microsoft Foundation Class (MFC) Library.
+- [Visual C++ 2013 Runtime for Sideloaded Windows 8.1 apps](http://download.microsoft.com/download/5/f/0/5f0f8404-9329-44a9-8176-ed6f7f746f25/vclibs_redist_packages.zip). For more information, see [C++ Runtime for Sideloaded Windows 8.1 apps] on the [VC++ Team Blog](https://devblogs.microsoft.com/cppblog/c-runtime-for-sideloaded-windows-8-1-apps/).
+
+## Visual Studio 2012 (VC++ 11.0) Update 4
+
+Below are the latest supported en-US Microsoft Visual C++ Redistributable packages for Visual Studio 2012 Update 4.
+Download additional versions and languages from [Microsoft Visual C++ Redistributable Packages for Visual Studio 2012 Update 4](https://www.microsoft.com/en-us/download/details.aspx?id=30679) or from [My.VisualStudio.com](https://my.visualstudio.com/Downloads?pid=1452).
+
+| Architecture |   Version    |                                                                                                                                                                                       Link |
+| ------------ | :----------: | -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| X86          | 11.0.61030.0 | [en_visual_cpp_redistributable_for_visual_studio_2012_update_4_x86_3161523.exe](https://download.microsoft.com/download/1/6/B/16B06F60-3B20-4FF2-B699-5E9B7962F9AE/VSU_4/vcredist_x86.exe) |
+| X64          | 11.0.61030.0 | [en_visual_cpp_redistributable_for_visual_studio_2012_update_4_x64_3161523.exe](https://download.microsoft.com/download/1/6/B/16B06F60-3B20-4FF2-B699-5E9B7962F9AE/VSU_4/vcredist_x64.exe) |
+
+## Visual Studio 2010 (VC++ 10.0) SP1 (no longer supported)
+
+Visual Studio 2010 Service Pack 1 [reached end of extended support on July 14, 2020](https://docs.microsoft.com/en-us/lifecycle/products/visual-studio-2010). To make it easy to find the latest downloads, the links below currently work, but may be removed in the future.
```

Comment:
All instances of 'currently work' should be removed because whenever someone reads this, it is 'currently'. Just list the link.

Reply:
Removed, and added the expiration of each as a "note" to highlight it.